Library/Homebrew/requirements/gpg_requirement.rb

```diff
@@ -0,0 +1,27 @@
+require "requirement"
+
+class GPGRequirement < Requirement
+  fatal true
+  default_formula "gnupg2"
+
+  satisfy(:build_env => false) { gpg2 || gpg }
+
+  # MacGPG2/GPGTools installs GnuPG 2.0.x as a vanilla `gpg` symlink
+  # pointing to `gpg2`, as do we. Ensure we're actually using a 2.0 `gpg`.
+  # Temporarily, only support 2.0.x rather than the 2.1.x "modern" series.
+  def gpg
+    which_all("gpg").detect do |gpg|
+      gpg_short_version = Utils.popen_read(gpg, "--version")[/\d\.\d/, 0]
+      next unless gpg_short_version
+      Version.create(gpg_short_version.to_s) == Version.create("2.0")
+    end
+  end
+
+  def gpg2
```

Comment:
I'm a bit confused about these methods; they are both looking for `gpg 2.0` but just looking with different names? If so, would be good to make the requirement named `GPG2Requirement` and share code between the two methods (as their only difference as far as I can see is the `which_all` parameter).